go/ct/rlz/expressions.go

```diff
@@ -160,6 +160,42 @@ func (gas) String() string {
 	return "Gas"
 }
 
+////////////////////////////////////////////////////////////
+// Self - the address of the called contract
+
+type self struct{}
```

Comment:
I think just `self` is not enough to convey to which part of the whole `self` is this expression referring to, self-balance, self-address, self-nonce, etc.

Reply:
I agree, self is not a good name for any type.

Reply:
Please propose an alternative.

Reply:
Sorry, I forgot to fix this, will add this fix and update PR after lunch.